internal/utils/nodeutils.go

```diff
@@ -12,3 +16,18 @@ func IsNodeSchedulable(node *v1.Node) bool {
 	}
 	return true
 }
+
+func IsObjectSelectedByLabels(objectLabels map[string]string, selectorLabels map[string]string) (bool, error) {
```

Comment:
currently there are not other options, unless we create a new file. Since anyway everything is under utils package, maybe we can rename the file to something more general?